src/main/java/lotto/step2/domain/WinningNumbers.java

```diff
@@ -0,0 +1,37 @@
+package lotto.step2.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class WinningNumbers {
+    private final List<Integer> winningNumbers;
+
+    public List<Integer> getWinningNumbers() {
+        return winningNumbers;
+    }
+
+    public WinningNumbers(List<Integer> winningNumbers) {
+        this.winningNumbers = validateWinningNumbers(winningNumbers);
+    }
+
+    private List<Integer> validateWinningNumbers(List<Integer> winningNumbers) {
+        if (winningNumbers.size() != 6) {
+            throw new IllegalArgumentException("당첨 번호는 6개만 가능합니다.");
+        }
+        if (Collections.max(winningNumbers) > 45 || Collections.min(winningNumbers) < 1) {
+            throw new IllegalArgumentException("당첨 번호는 1 ~ 45까지만 가능합니다.");
+        }
+        if (winningNumbers.stream().distinct().count() != 6) {
```

Comment:
이런 로직은 하나의 분기문 블럭안에 넣기보단 의미있는 네이밍으로 분리해주면 코드의 가독성이 높아질 것 같아요. 

```suggestion
        if (size() != 6) {
```